src/styles/_variables.scss

```
@@ -0,0 +1,9 @@
+$main-bg: #eee;
+$hover-bg: #ffbfcb;
+$size: 100px;
+$font: Arial, sans-serif;
+$font-size: 30px;
+$hover-offset: 20px;
+$gap: 1px;
+$padding: 10px;
+$border: 1px solid black;
```

u don't need a variables for it. don't create normal variables for fonts. moreover - u use it only once
```suggestion
$gap: 1px;
$padding: 10px;
```